threedi_scenario_downloader/downloader.py

```diff
@@ -339,11 +346,25 @@ def transform_to_list(var, length=1):
             if type(scenario) is str:
                 # assume uuid
                 scenario_instance = get_scenario_instance(scenario)
-                raster = get_raster(scenario, raster_code)
+                if raster_code == "depth-first-dtri":
+                    raster = get_raster(scenario, raster_code, subendpoint="arrival")
+                elif raster_code == "total-damage":
+                    raster = get_raster(scenario, raster_code, subendpoint="damage")
+                else:
+                    raster = get_raster(scenario, raster_code)
             elif type(scenario) is dict:
                 # assume json object
                 scenario_instance = scenario
-                raster = get_raster_from_json(scenario, raster_code)
+                if raster_code == "depth-first-dtri":
+                    raster = get_raster_from_json(
+                        scenario, raster_code, subendpoint="arrival"
+                    )
+                elif raster_code == "total-damage":
+                    raster = get_raster_from_json(
+                        scenario, raster_code, subendpoint="damage"
+                    )
+                else:
+                    raster = get_raster_from_json(scenario, raster_code)
```

Comment:
Dit is drie keer hetzelfde (want de laatste is eigenlijk `subendpoint=None`. Dit is eleganter en leesbaarder en korter:

```suggestion
                subendpoint_per_raster_code = {
                   "depth-first_dtri": "arrival",
                   "total-damage": "damage",
                }
                subendpoint = subendpoint_per_raster_code.get(raster_code)
                raster = get_raster_from_json(
                    scenario, raster_code, subendpoint=subendpoint)
                )
```